Topics/Tech_Stacks/Iframe.md

```diff
@@ -0,0 +1,25 @@
+# Learning Iframe
+
+## Introduction
+
+An Iframe is a html element that can let programmer to display a child website inside a parent website. It is supported by many browsers, like chrome, firefox and safari.
+
+## Why use Iframe
+Sometimes we want to let user to interact with a webpage without open a new webpage tage and redict to it. For example, we want to embed a survey inside a webpage so that the user can conveniently fill out the survey after reading the previous content in this webpage.
+
+## How to use Iframe
+You can add a html element inside your html file following this format <iframe src="some like" title="some title"></iframe>
+
+There is two important things related to Iframe and I provide instructions about them
+
+### Dynamically resizing an iframe based on content
+You can use get the scrollhight of the child website by add a javascript block in the parent html file which contains X.contentWindow.document.body.scrollHeight + 'px';
```

Comment:
Lots of typos here. Also, please wrap your code snippets in ``.

- "use get"
- "scrollhight"
- "by add"